src/handlers/episode_table.rs

```diff
@@ -0,0 +1,96 @@
+use super::{
+  super::app::{App, EpisodeTable},
+  common_key_events,
+};
+use crate::event::Key;
+use crate::network::IoEvent;
+
+pub fn handler(key: Key, app: &mut App) {
+  match key {
+    k if common_key_events::left_event(k) => common_key_events::handle_left_event(app),
+    k if common_key_events::down_event(k) => {
+      let next_index = common_key_events::on_down_press_handler(
+        &app.episode_table.episodes,
+        Some(app.episode_table.selected_index),
+      );
+      app.episode_table.selected_index = next_index;
+    }
+    k if common_key_events::up_event(k) => {
+      let next_index = common_key_events::on_up_press_handler(
+        &app.episode_table.episodes,
+        Some(app.episode_table.selected_index),
+      );
+      app.episode_table.selected_index = next_index;
+    }
+    k if common_key_events::high_event(k) => {
+      let next_index = common_key_events::on_high_press_handler();
+      app.episode_table.selected_index = next_index;
+    }
+    k if common_key_events::middle_event(k) => {
+      let next_index = common_key_events::on_middle_press_handler(&app.episode_table.episodes);
+      app.episode_table.selected_index = next_index;
+    }
+    k if common_key_events::low_event(k) => {
+      let next_index = common_key_events::on_low_press_handler(&app.episode_table.episodes);
+      app.episode_table.selected_index = next_index;
+    }
+    Key::Enter => {
+      on_enter(app);
+    }
+    // Scroll down
+    Key::Ctrl('d') => {}
```

Comment:
Would be great if the pagination keys referenced the new keys from the user config (` app.user_config.keys.next_page` as implemented here https://github.com/Rigellute/spotify-tui/pull/566).

Not a blocker for this PR though since there is no pagination yet.

Reply:
Good catch. I agree that this would probably get fixed when paging gets implemented for this table, but I'll pop that in now just so it doesn't get overlooked.

Reply:
Should be fixed now.